Formula/libtommath.rb

```diff
@@ -28,10 +30,10 @@ class Libtommath < Formula
   def install
     ENV["DESTDIR"] = prefix
 
-    system "make"
-    system "make", "test_standalone"
+    system "make", "-f", "makefile.shared"
+    system "make", "test"
     include.install Dir["tommath*.h"]
-    lib.install "libtommath.a"
+    lib.install Dir[".libs/libtommath*{.a,.dylib}"]
```

Comment:
```suggestion
    lib.install Dir[".libs/libtommath*.a"]
    lib.install Dir["./libs/#{shared_library("libtommath", "*")}"]
```
I think this should make it portable for Linuxbrew. Oh, and can you increment the `revision` to `2`? Squash your commits together (if you have multiple) when you're done. Thanks!